gherkin-tests/src/world.rs

```diff
@@ -0,0 +1,114 @@
+use std::{collections::BTreeMap, convert::Infallible, str::FromStr, sync::Arc};
+
+use async_trait::async_trait;
+use cucumber::{Parameter, WorldInit};
+use many_client::client::ledger::{LedgerClient, Symbol, TokenAmount};
+use many_client::ManyClient;
+use many_identity::{Address, CoseKeyIdentity};
+
+use crate::{cose::new_identity, opts::SpecConfig};
+
+#[derive(Parameter, Debug, Hash, Ord, PartialOrd, Eq, PartialEq)]
+#[param(regex = r"[\w\d]+", name = "identity")]
+pub struct IdentityName(String);
```

Comment:
Like I said above, let's name this `identifier` and reuse it for symbols as well.